airflow/utils/db.py

```diff
@@ -887,9 +889,9 @@ def _check_migration_errors(session=None) -> Iterable[str]:
         check_run_id_null,
         check_task_tables_without_matching_dagruns,
     ):
-        yield from check_fn(session)
+        yield from check_fn(session)  # type: ignore
         # Ensure there is no "active" transaction. Seems odd, but without this MSSQL can hang
-        session.commit()
+        session.commit()  # type: ignore
```

Comment:
We should fix this in `airflow.settings` instead, it’d be a nightmare dealing with all these ignores for every `session` call.

Reply:
True

Reply:
For now in db I solve it differently. But maybe we will figure out to solve it differently globally via the annotation. Any ide how this can be done?